pkg/apis/core/types.go

```diff
@@ -1851,6 +1851,13 @@ type PodCertificateProjection struct {
 	// Kubelet's generated CSRs will be addressed to this signer.
 	SignerName string
 
+	// User-requested configuration that is copied directly into the PodCertificateRequest.
+	// Keys must be namespaced in the form `<domainName>/<keyName`>.  Signer
+	// implementations should document what UserConfig keys they support. Signer
+	// implementations should reject PodCertificateRequests that contain user config that
+	// they do not recognize with a condition of type "Denied" and reason "InvalidUserConfig".
+	UserConfig map[string]string
```

Comment:
This was renamed to `UserAnnotations`.  No `Untrusted...`, that's only on the PodCertificateRequest object.